apps/docs/scripts/typedoc-postbuild.ts

```diff
@@ -174,4 +174,4 @@ const main = async () => {
   recreateInternalLinks();
 };
 
-main();
+main().catch(process.stderr.write);
```

Comment:
What was this in aid of?

Reply:
After turning on [this](https://github.com/FuelLabs/fuels-ts/pull/1228/commits/8121acbc5320024e2d67ec7f2ea3f1783d12021a) `eslit` rule, I had to fix all unhandled promises, and this was one of them.

You can see it all in this commit. https://github.com/FuelLabs/fuels-ts/pull/1228/commits/83e954f3ddf7acb6e5e96d68ed107d377584c302